src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -719,13 +704,42 @@ inline void rpp_load48_i8pkd3_to_i8pln3(Rpp8s *srcPtr, __m128i *px)
     px[2] = _mm_shuffle_epi8(_mm_unpacklo_epi8(pxSrc[6], pxSrc[7]), pxMaskRGB);    /* unpack 8 lo-pixels of pxSrc[6] and pxSrc[7] to get B01-16 */
 }
 
+inline void rpp_load48_i8pln3_to_i32pln3_avx(Rpp8s *srcPtrR, Rpp8s *srcPtrG, Rpp8s *srcPtrB, __m256i *p)
```

Comment:
I intially felt its better to have this with pln3_to_pln3 instead of pkd3_to_pln3, anyways none of the loads are in proper order so reverting this change.